lib/common/exceptions/external/backend_service_exception.rb

```diff
@@ -65,25 +79,20 @@ def code
       end
 
       # REQUIRED - This is the http status code.
-      # unless you've specified that you want the status code to be something other
-      # then 400 explicitly it will default to 400. IT WILL NOT DEFAULT to whatever
-      # was provided by the backend service, because the backend service response
-      # might not always be relevant
       def status
-        i18n_data[:status].presence || 400
+        i18n_data[:status] || 400
```

Comment:
You can take a look at how we use it in raise_error.rb middleware and how mhv_errors.rb middleware coerces the errors that MHV returns. In addition we have a mhv_xml_html_errors.rb middleware that handles 500 level responses returned from MHV.

With this refactor, it should be more easy than ever to leverage BackendServiceException, especially with the new 500 level errors defined, specifically VA1003 and VA1002 which correspond to 503 and 500 respectively.